tests/fixtures/valid_analysis/policies/example_policy_extraneous_fields.yml

```diff
@@ -1,5 +1,5 @@
 AnalysisType: policy
-Filename: example_policy.py
+Filename: example_policy_extraneous_fields.py
```

Comment:
This was causing a name conflict with some annoying output in the zipping unit test, so I just duplicated the policy body  into  a new name. Hopefully  this doesn't go against the purpose of the test.